lib/reddex/reports/report.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Reddex.Report do
+  alias Reddex.Links
+  alias Reddex.ReportItem
+
+  def send do
+    generate()
+    |> Enum.each(&send_report_item/1)
+  end
+
+  def generate do
```

Comment:
I would rename this function, because you don't generate anything here. You select and format the links.